install.sh

```diff
@@ -45,21 +45,28 @@ MONDOO_BINARY="cnspec" # binary that we search for
 
 # read bash flags
 MONDOO_INSTALLER=''
+MONDOO_SERVICE=''
 MONDOO_REGISTRATION_TOKEN=''
 
 print_usage() {
   echo "usage: [-i]" >&2
   echo "  Options: " >&2
   echo "    -i <installer>:  Select a specific installer, options are:" >&2
   echo "                     macOS: brew, pkg" >&2
+  echo "    -s <service>:    Enables the cnspec service for the system." >&2
+  echo "                     options are: enable" >&2
```

Comment:
@atomic111 is there an option for `disable` as well? if not, why do we have options? Why not just `-s` enables cnspec to run as a service?

Reply:
i want to create the disable option later